lix/packages/sdk/src/database/schema.ts

```diff
@@ -4,8 +4,9 @@ import type { LixPlugin } from "../plugin.js";
 export type LixDatabaseSchema = {
 	file: LixFileTable;
 	change: ChangeTable;
-	commit: CommitTable;
-	ref: RefTable;
+	change_view: ChangeView;
```

Comment:
if we remove the commit function - fink and so on will fail with this branch - shall we make the required changes in fink, csv app etc part of this brach?

Reply:
as discussed, yes we do!